src/main/java/codeu/model/data/User.java

```diff
@@ -61,6 +61,12 @@ public Instant getCreationTime() {
 
   @Override
   public int compareTo(User otherUser) {
-    return getCreationTime().compareTo(otherUser.getCreationTime());
+    if(otherUser.equals(null) || this.equals(null))
```

Comment:
this can't be null so the second part isn't necessary. Also a null comparison doesn't require per member comparison so you can use "otherUser == null" instead.